### ISSUE TITLE
Flow card with temperature forecast

### ISSUE THREAD

**Issue by Melbo82** [Inversion-NL/nl.inversion.wunderground#38]
Can you add a flow card with temperature forecast? 

For example: 
I would like my window covers to stay down in the morning when it will be warm in the afternoon so I can keep my home nice and cold! :) 

Could you add this feature? Thanks!

**Comment by Inversion-NL** [comment]
Been thinking about this.
I guess it would be as an condition card.
Then you could use 'Every hour' or at '8:00 am' and 'WU nice weather' then close blinds.

Because when you use it as an action card (In the most left column, in the When column) and some users update WU could be every 10min, it would trigger every 10 min.
Which will be not very resource friendly for Homey's hardware.

Let me know what you think.

**Comment by Melbo82** [comment]
Thanks for your quick reply.

I was thinking to make a flow which checks the temperature forecast for
that day at 8:00 am. If the forecasted temperature is more then 21 degrees
celcius then Homey sets itself to ‘summer mode’. In that mode Homey manages
the blinds to keep the house cold. If the forecasted temperature is less
then 21 degrees Homey sets itself to ‘standard mode’ and then manages the
blinds to get as much light in the house as possible.

Is it possible to make a flowcard for the ‘and’ column with which you can
trigger on forecasts for that day? For me temperature forecast would be ok
but I can image that there are people who could use other forecasts as well.

Regards,
Melvin



2017-08-21 14:09 GMT+02:00 Inversion-NL <notifications@github.com>:

**Comment by Inversion-NL** [comment]
Couldn't let it go and started a little investigation.
The data I get is for every hour: temperature and condition.
Temperature is no problem.
Condition, however, is potentially an problem: I don't know all conditions and there isn't an list of all available conditions.

**Comment by Melbo82** [comment]
I understand the problem with the weather forecast. Maybe we can try out the temperature forecast first and see how that holds up in practise?

Regards,
Melvin

Op 21 aug. 2017 om 16:53 heeft Inversion-NL <notifications@github.com> het volgende geschreven:

Couldn't let it go and started a little investigation.
The data I get is for every hour: temperature and condition.
Temperature is no problem.
Condition, however, is potentially an problem: I don't know all conditions and there isn't an list of all available conditions.

**Comment by Inversion-NL** [comment]
Yes, I'll register forecast data as global tokens so you'll be able to use them anywhere in your flows!
Even in an action card for voice output, mail and any other cards which accepts tokens.

**Comment by Melbo82** [comment]
Great, looking forward to it! :)

Thanks for your great support on this app!

Regards,
Melvin

Op 21 aug. 2017 om 17:24 heeft Inversion-NL <notifications@github.com> het volgende geschreven:

Yes, I'll register forecast data as global tokens so you'll be able to use them anywhere in your flows!
Even in an action card for voice output, mail and any other cards which accepts tokens.

**Comment by Inversion-NL** [comment]
If all goes well and I don't run in to any big issues while finishing the mobile card I'll have this feature implemented tonight.
Are you willing to run the beta?
If you want to go back to the stable you'll have to uninstall this app (and lose all Insights history along with it)

**Comment by Melbo82** [comment]
Sure, no problem. 

What do I have to do to run the Beta?

Regards,
Melvin

Op 21 aug. 2017 om 18:32 heeft Inversion-NL <notifications@github.com> het volgende geschreven:

If all goes well and I don't run in to any big issues while finishing the mobile card I'll have this feature implemented tonight.
Are you willing to run the beta?
If you want to go back to the stable you'll have to uninstall this app (and lose all Insights history along with it)

**Comment by Inversion-NL** [comment]
Wait for me to publish it to the beta ;-)

**Comment by Melbo82** [comment]
Allright, let me know when you have the Beta ready and then I'll go on and test it! ;)

Regards,
Melvin


Op 22 aug. 2017 om 00:49 heeft Inversion-NL <notifications@github.com> het volgende geschreven:

Wait for me to publish it to the beta ;-)

**Comment by Melbo82** [comment]
Hi there,

Made any progess with the Beta? 😊 

Regards,
Melvin

Op 22 aug. 2017 om 09:06 heeft Melvin de Vos <melvin@dvos.eu> het volgende geschreven:

Allright, let me know when you have the Beta ready and then I'll go on and test it! ;)

Regards,
Melvin


Op 22 aug. 2017 om 00:49 heeft Inversion-NL <notifications@github.com> het volgende geschreven:

Wait for me to publish it to the beta ;-)
